EpsilonAuxBMS/Src/UpdateChargeAllowanceTask.c

```diff
@@ -20,26 +20,100 @@ void updateChargeAllowanceTask(void const* arg)
            and the cell with the lowest voltage. Must check if the voltage of the cell with the highest
            voltage is too high or in range and if the cell with the lowest voltage is too low or in range.
         */
+        if (orionStatus.shutOff)
+        {
+            continue;
+        }
 
         uint8_t voltagesInRange = 1;
         uint8_t allowCharge = 1;
         uint8_t allowDischarge = 1;
+        uint8_t chargeContactorOverride = 0;
+        uint8_t dischargeContactorOverride = 0;
+        uint8_t commonContactorOff = 0;
+        uint8_t shutOff = 0;
 
-        if (DEFAULT_VOLTAGE_UNITS * orionStatus.maxCellVoltage > MAX_CELL_VOLTAGE * 0.8) // Will have the cutoff to be 20% below
+        if (auxStatus.startUpSequenceDone)
```

Comment:
We should make a comment explaining how each input affects the board in each mode.